File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def main():
    height = 0
    while height < 1 or height > 8:  # get correct range for height
        height = get_int("Height: ")
    first_pyramid(height)


def first_pyramid(height):  # print out pyramid
    for i in range(height):
        print(" " * (height - i - 1), end="")
        print("#" * (i + 1), end="")
        print(" " * 2, end="")
        print("#" * (i + 1))


if __name__ == "__main__":
    main()

File: cs50x/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get the height from user
    int height = 0;
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
         
    }


    // print out # pyramid
    for (int i = 1; i < height + 1; i++)
    {
        // print out spaces
        for (int k = height; k > i; k--)
        {
            printf(" ");
        }

        //print out #
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");

    }
}